syntax = "proto3";

package pose_classifier;

import "pose.proto";

message ClassificationRequest {
  pose.PoseReading pose = 1;
}

message ClassificationResponse {
  bool is_upside_down = 1;
  pose.PoseReading source_pose = 2;
}
